src/views/pam/RiskDetect/AccountCheckExecutionList.vue

```diff
@@ -63,7 +63,7 @@ export default {
               drawer: true,
               getTitle: ({ row }) => row.snapshot.name,
               getRoute: ({ row }) => ({
-                name: 'AccountCheckList',
+                name: 'AccountCheckDetail',
                 params: { id: row.automation }
               })
             },
```

Comment:
The code seems to be well written and doesn't contain any major issues or irregularities that could need further improvements. No changes suggested on this line of development at all.
No knowledge cutoff necessary.